examples/question_the_docs copy.ipynb

```diff
@@ -0,0 +1,3283 @@
42b3f43aad980e839ad4'), ObjectId('656b42b3f43aad980e839ad5'), ObjectId('656b42b3f43aad980e839ad6'), ObjectId('656b42b3f43aad980e839ad7'), ObjectId('656b42b3f43aad980e839ad8'), ObjectId('656b42b3f43aad980e839ad9'), ObjectId('656b42b3f43aad980e839ada'), ObjectId('656b42b3f43aad980e839adb'), ObjectId('656b42b3f43aad980e839adc'), ObjectId('656b42b3f43aad980e839add'), ObjectId('656b42b3f43aad980e839ade'), ObjectId('656b42b3f43aad980e839adf'), ObjectId('656b42b3f43aad980e839ae0'), ObjectId('656b42b3f43aad980e839ae1'), ObjectId('656b42b3f43aad980e839ae2'), ObjectId('656b42b3f43aad980e839ae3'), ObjectId('656b42b3f43aad980e839ae4'), ObjectId('656b42b3f43aad980e839ae5'), ObjectId('656b42b3f43aad980e839ae6'), ObjectId('656b42b3f43aad980e839ae7'), ObjectId('656b42b3f43aad980e839ae8'), ObjectId('656b42b3f43aad980e839ae9'), ObjectId('656b42b3f43aad980e839aea'), ObjectId('656b42b3f43aad980e839aeb'), ObjectId('656b42b3f43aad980e839aec'), ObjectId('656b42b3f43aad980e839aed'), ObjectId('656b42b3f43aad980e839aee'), ObjectId('656b42b3f43aad980e839aef'), ObjectId('656b42b3f43aad980e839af0'), ObjectId('656b42b3f43aad980e839af1'), ObjectId('656b42b3f43aad980e839af2'), ObjectId('656b42b3f43aad980e839af3'), ObjectId('656b42b3f43aad980e839af4'), ObjectId('656b42b3f43aad980e839af5'), ObjectId('656b42b3f43aad980e839af6'), ObjectId('656b42b3f43aad980e839af7'), ObjectId('656b42b3f43aad980e839af8'), ObjectId('656b42b3f43aad980e839af9'), ObjectId('656b42b3f43aad980e839afa'), ObjectId('656b42b3f43aad980e839afb'), ObjectId('656b42b3f43aad980e839afc'), ObjectId('656b42b3f43aad980e839afd'), ObjectId('656b42b3f43aad980e839afe'), ObjectId('656b42b3f43aad980e839aff'), ObjectId('656b42b3f43aad980e839b00'), ObjectId('656b42b3f43aad980e839b01'), ObjectId('656b42b3f43aad980e839b02'), ObjectId('656b42b3f43aad980e839b03'), ObjectId('656b42b3f43aad980e839b04'), ObjectId('656b42b3f43aad980e839b05'), ObjectId('656b42b3f43aad980e839b06'), ObjectId('656b42b3f43aad980e839b07'), ObjectId('656b42b3f43aad980e839b08'), ObjectId('656b42b3f43aad980e839b09'), ObjectId('656b42b3f43aad980e839b0a'), ObjectId('656b42b3f43aad980e839b0b'), ObjectId('656b42b3f43aad980e839b0c'), ObjectId('656b42b3f43aad980e839b0d'), ObjectId('656b42b3f43aad980e839b0e'), ObjectId('656b42b3f43aad980e839b0f'), ObjectId('656b42b3f43aad980e839b10'), ObjectId('656b42b3f43aad980e839b11'), ObjectId('656b42b3f43aad980e839b12'), ObjectId('656b42b3f43aad980e839b13'), ObjectId('656b42b3f43aad980e839b14'), ObjectId('656b42b3f43aad980e839b15'), ObjectId('656b42b3f43aad980e839b16'), ObjectId('656b42b3f43aad980e839b17'), ObjectId('656b42b3f43aad980e839b18'), ObjectId('656b42b3f43aad980e839b19'), ObjectId('656b42b3f43aad980e839b1a'), ObjectId('656b42b3f43aad980e839b1b'), ObjectId('656b42b3f43aad980e839b1c'), ObjectId('656b42b3f43aad980e839b1d'), ObjectId('656b42b3f43aad980e839b1e'), ObjectId('656b42b3f43aad980e839b1f'), ObjectId('656b42b3f43aad980e839b20'), ObjectId('656b42b3f43aad980e839b21'), ObjectId('656b42b3f43aad980e839b22')]\u001b[0m\n",
```

Comment:
We can only keep the important output. I think other output can be deleted.

And remove the debug log

Reply:
Important output will be gone too if I remove this part. Attached with debug log!

Reply:
Can we set the logging level to do this?